### PR TITLE
test(webiss): add comprehensive XML tests with DpsDocumentBuilder + E2E

### LINKED ISSUE #64: test(webiss): cobertura completa de geração XML com validação XSD e cenários de preenchimento
## Contexto
O provider **WebISS** é ABRASF-based. Status: SupportReady. Sem artefatos comparativos (não há XSD validation report).

## Problema atual
Sem testes dedicados. Sem validação XSD confirmada. Sem cenários de preenchimento cobertos.

## Objetivo
Cobertura completa de geração XML com validação XSD.

### Testes de integração
- [ ] Cenário mínimo → XML valida XSD
- [ ] Cenário completo → XML valida XSD
- [ ] Variações de TaxationType
- [ ] Borrower PJ e PF
- [ ] Grupos opcionais ABRASF
- [ ] Envelope ABRASF correto

### Testes unitários
- [ ] Regras do provider testadas
- [ ] Edge cases

### Validação XSD
- [ ] Todo cenário valida XML contra XSD do WebISS

## Dependências
Nenhuma

## Estimativa
M (médio)

### PR DESCRIPTION
**Pull request by john182**
28 unit tests + 2 E2E integration tests. Closes #64.

### CODE REVIEW

File: tests/SemanaIA.ServiceInvoice.UnitTests/SchemaEngine/WebissXmlSerializationTests.cs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        schema.ComplexTypes.Count.ShouldBeGreaterThan(0);
+        schema.TargetNamespace.ShouldNotBeNullOrEmpty();
+    }
+
+    [Fact]
+    public void Given_WebissProvider_Should_DetectEnvelopeAndRootElement()
+    {
+        var config = new ProviderConfigGenerator(TestProviderPaths.FindProvidersDir()).GenerateConfig(ProviderName);
+        config.ShouldNotBeNull();
+        config.Provider.ShouldBe(ProviderName);
```
[the remark this answers is not in the document]
Comment (john182) [reply tests/SemanaIA.ServiceInvoice.UnitTests/SchemaEngine/WebissXmlSerializationTests.cs]: Resolved in latest commit.

**Comment by john182** [comment]
@github-copilot re-review — all 5 comments addressed
